tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+cookiecutter==0.9.0
+coverage==3.7.1
+ddt==0.8.1
+diff-cover==0.7.2
+Django==1.4.16
+django_nose==1.2
+fs==0.5.0
+lazy==1.2
+mock==1.0.1
+nose==1.3.4
+nose-parameterized==0.3.5
+pep8==1.5.7
+pylint==0.28
+pypng==0.0.17
+rednose==0.4.1
+requests==2.4.3
+selenium==2.44.0
+simplejson==3.6.5
+unicodecsv==0.9.4
+webob==1.4
```

Comment:
@mtyaka I still don't think *all* of this is required: e.g. there are no direct dependencies on Django, pep8 and pylint are not even listed in readme, pypng is unlikely to be used as well, etc.. I would suggest trying removing **from this file** dependencies that are dependencies of xblock-sdk or xblock-utils or other direct dependencies => they will be automatically installed along with sdk, utils, etc., and when they are updated there we'll have a conflict here.